cmd/auth-vue/vue.config.js

```diff
@@ -0,0 +1,9 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+module.exports = {
+    publicPath: "/ui"
+}
```

Comment:
Another no-newline warning

Reply:
eslint works differently then the golang one.